src/app/containers/FrontPageSection/index.jsx

```diff
@@ -90,18 +84,28 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
     // the greatest possible support.
     // eslint-disable-next-line jsx-a11y/no-redundant-roles
     <section role="region" aria-labelledby={sectionLabelId}>
-      <SectionLabel
-        script={script}
-        labelId={sectionLabelId}
-        bar={bar}
-        visuallyHidden={isFirstSection}
-        service={service}
-        dir={dir}
-        linkText={isLink ? seeAll : null}
-        href={href}
-      >
-        {group.strapline.name}
-      </SectionLabel>
+      {/*
+         The current implementation of SectionLabel *requires* a strapline to be
+         present in order to render. It is currently *not possible* to render a
+         section that does not have a strapline without breaking both the visual
+         *and especially* the screen reader UX.
+         If this group does not have a strapline; do not render!
+         This may change in the future, if a way to avoid breaking UX is found.
+      */}
+      {strapline && (
```

Comment:
```The current implementation of SectionLabel *requires* a strapline to be present in order to render.```
This is no longer true. but IMO this shouldnt be changed. We need Section Label 100% of the time for A11y

Reply:
In cases where we have no `strapline` like in feature-indexes, what should be used as the `name`. I can see we are displaying `group.strapline.name`.

Reply:
I am checking now if straplines are supported/required in feature index. if they are then for now we can just modify the fixture data

Reply:
Straplines are support, but right now as they do not render on our legacy pages many FIX assets do not have them.

Example test with a strapline `/pidgin/tori-23250077`

Reply:
Awesome, sounds like the fixture data needs updating, and this logic can be reverted @sadick254

Reply:
@dr3 Done